Makefile

```diff
@@ -21,9 +21,9 @@ ifeq ($(shell $(GO) env GOOS),linux)
 	endif
 endif
 GO_BUILD := $(GO) build -trimpath $(GO_BUILDMODE) $(EXTRA_FLAGS) -tags "$(BUILDTAGS)" \
-	-ldflags "-X main.gitCommit=$(COMMIT) -X main.version=$(VERSION) $(EXTRA_LDFLAGS)"
+	-ldflags "-X main.gitCommit=$(COMMIT) $(EXTRA_LDFLAGS)"
 GO_BUILD_STATIC := CGO_ENABLED=1 $(GO) build -trimpath $(EXTRA_FLAGS) -tags "$(BUILDTAGS) netgo osusergo" \
-	-ldflags "-extldflags -static -X main.gitCommit=$(COMMIT) -X main.version=$(VERSION) $(EXTRA_LDFLAGS)"
+	-ldflags "-extldflags -static -X main.gitCommit=$(COMMIT) $(EXTRA_LDFLAGS)"
```

Comment:
Note that Go 1.18 will also embed VCS info (and other details), which means we may be able to get rid of the `-X main.gitCommit` as well; https://github.com/golang/go/issues/37475 was accepted (more details in https://github.com/golang/go/issues/35667)

Reply:
Alas, the way it's done by Go toolchain is not as good looking as we do:

```console
[kir@kir-rhat runc]$ ./runc --version
runc version 1.1.0+dev
commit: v1.1.0-710-gbaf464eb # <<<<<
spec: 1.1.0
go: go1.20.7
libseccomp: 2.5.3
[kir@kir-rhat runc]$ go version -m ./runc | grep vcs
	build	vcs=git
	build	vcs.revision=baf464eb1ace0fbecb3c4671bb5e19a838ec9d18 # <<<<<
	build	vcs.time=2023-08-11T19:22:52Z
	build	vcs.modified=true
```

Both show the sha, but we also show the last annotated tag tag and the distance from it (number of commits).

Reply:
Also, I've had issues with the Go VCS embedding before in umoci (for some reason, it can lead to non-deterministic builds).